circle.yml

```diff
@@ -180,10 +180,14 @@ commands:
     steps:
       - run:
           name: Build all codegen
-          command: yarn gulp buildProd
+          command: |
+            source ./scripts/ensure-node.sh
```

Comment:
The default node on the image is 16.14.0; now that we're on 16.14.2, we're seeing all the areas in which we weren't properly sourcing nvm before. Hence all the ensure-node calls. Prior to this, our `>=16.13.2` resolution would work just fine. I'd like to take a look and see how we can remove all these calls, a cross-architecture solution may be a little painful.